ui/admin/tests/unit/controllers/scopes/scope/credential-stores/credential-store/credentials/new-test.js

```diff
@@ -4,7 +4,7 @@
  */
 
 import { module, test } from 'qunit';
-import { setupTest } from 'ember-qunit';
+import { setupTest } from 'admin/tests/helpers';
```

Comment:
In a similar PR I saw we changed using `admin/test/helpers` in favour of `ember-qunit`, why here we are doing the opposite?

Reply:
Ah yes, i saw the comment about this and responded [here](https://github.com/hashicorp/boundary-ui/pull/2176#discussion_r1523928351)

Reply:
Gootcha! answered on your respond, this is something I wasn't aware of it, perhaps we should discuss it as a team 😉